### ISSUE TITLE
State of the project?

### ISSUE THREAD

**Issue by yochananmarqos** [clipto-pro/Desktop#156]
@atrashler You've been MIA for two or three years. Anybody home?

**Comment by herrmann0319** [comment]
I spoke to him several years ago on Reddit and at that time he had no intention of updating or working on the app any longer bc he felt it was complete. What a shame. He should pass the code on to another developer or make it open source. It's such a good app. What a disssapointment...

**Comment by dead5eas** [comment]
It at least needs maintenance... new bugs to squash every Android version. Even the most passive income requires a little work if you want to keep the users/memberships flowing, right?

**Comment by herrmann0319** [comment]
Correct. I guess he just thought this one would be a set it and forget it, lol. I'm just so surprised because the amount of effort he clearly put into this thing you would think there would be some level of passion attached to it.

I tried messaging him on Reddit again and I'm going to create a post on his sub reddit to hopefully make him aware. I tried doing it the other day but it wasn't allowing me to hit post for some reason.

**Comment by herrmann0319** [comment]
> It at least needs maintenance... new bugs to squash every Android version. Even the most passive income requires a little work if you want to keep the users/memberships flowing, right?

Well soo much for keeping the memberships flowing, lol. This thing isn't even on the Play Store anymore and it looks like my notes aren't syncing as the .99c month plan is no longer available. Yet I have still been getting charged for it as of April 18th. 

On the app it says sync is off due to reached limit of 300 and .99c is greyed out along with some of the other more affordable options. I wonder if it will charge this month? I would think not since it's charged through Google Play and it's not on there anymore. Well, I just did a test by trying to subscribe to $2.99 a month to see what happens and it says error. So his funds have been completely cut off. Hopefully this will have him running back. Unless he's dead?

I wonder what's going on? This developer really just shit the bed on one of the best apps I have ever downloaded in my life! I don't think I can think of a single app on my phone that screams the same amount of effort that was put into this. 

And he just abandons it? I'm pretty sure you have to manually remove an app off the Play Store unless they find some sort of violation. I wish this guy would open his mouth and SAY something, anything!! Lol.
